### PR TITLE
Remove travis_retry

### PR DESCRIPTION
**Pull request by limonte**
If one single test in one single platform is failing then all tests will be performed three times, which will cause a big queue in SauceLabs.

`travis_retry` is hiding the dust under the carpet, we need to write **STABLE AND RELIABLE** tests.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-element-skeleton/12)

### CODE REVIEW

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[.travis.yml, line 16 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-KaM0y0LI_uf9Vu8dzBc:-KaM0y0LI_uf9Vu8dzBd:bdynd0h) ([raw file](https://github.com/vaadin/vaadin-element-skeleton/blob/850fb5f21c1c483b3cb564ddd9951319ff98c47c/.travis.yml#L16)):*
> ```YAML
> script:
>   - gulp lint
>   - travis_retry xvfb-run -s '-screen 0 1024x768x24' wct -l chrome -l firefox
> ```

At the same time I would add the approach I have in the performance tests prototype.
Basically run all chrome tests in local before testing in SauceLabs. That makes fail the build earlier if there are stupid tests failing. It does not hits so much the overall time test, and would increase SauceLabs queue availability.

Take a look to this: https://github.com/vaadin/vaadin-context-menu/commit/4951fac5a6e51f54edf8489673b2be7ca8dad953#diff-354f30a63fb0907d4ad57269548329e3R16

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12)*

**Comment by manolo** [comment]
Thinking more about this change, we added the retry because we had many random failures , and we wanted to make sure that it wasn't a problem in tests but in the infra.
These random failures used to happen more in IE and mobile, I don't know if we have success builds because of this and how many that otherwise would be failed.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-:-KaM2EkWgCBEut0NwFOl:b-urmgxf)*

**Comment by limonte** [comment]
> random failures

This is my main point. There shouldn't be random failures on any platform.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[.travis.yml, line 16 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-KaM0y0LI_uf9Vu8dzBc:-KaM26FvR45L_fSUgoHe:bhg4wy2) ([raw file](https://github.com/vaadin/vaadin-element-skeleton/blob/850fb5f21c1c483b3cb564ddd9951319ff98c47c/.travis.yml#L16)):*
<details><summary><i>Previously, manolo (Manuel Carrasco Moñino) wrote…</i></summary><blockquote>

At the same time I would add the approach I have in the performance tests prototype.
Basically run all chrome tests in local before testing in SauceLabs. That makes fail the build earlier if there are stupid tests failing. It does not hits so much the overall time test, and would increase SauceLabs queue availability.

Take a look to this: https://github.com/vaadin/vaadin-context-menu/commit/4951fac5a6e51f54edf8489673b2be7ca8dad953#diff-354f30a63fb0907d4ad57269548329e3R16

</blockquote></details>

Travis will continue all tasks listed in `script` section of `.travis.yml`. So this make no sense to me. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-:-KaM2uYW6HlZy8HejFQQ:b-feduqb)*

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[.travis.yml, line 16 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-KaM0y0LI_uf9Vu8dzBc:-KaM3v1LP8avtLFTjKH-:b-8o3l7w) ([raw file](https://github.com/vaadin/vaadin-element-skeleton/blob/850fb5f21c1c483b3cb564ddd9951319ff98c47c/.travis.yml#L16)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

Travis will continue all tasks listed in `script` section of `.travis.yml`. So this make no sense to me. 
</blockquote></details>

Oh, I thought travis stops when there is a non zero exit code.
But we still have options, like run fast tests in the `before_script` phase ?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12)*

**Comment by manolo** [comment]
But it's something out of our control, you don't know when an android installation in saucelabs could be faulty. 
Obviously we must avoid any flaky test in our build.
I'm not against this change, in fact those retries are annoying when we already know that a test is failing. 
The point is whether we can have more false negatives with this change. Probably things have changed in SauceLabs platform and it produces less random failures

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-:-KaM4iqwFJI2BSgfDe91:b-eyp89q)*

**Comment by limonte** [comment]
Unfortunately, there are still errors like `Error: socket hang up` sometimes :( So, most probably you idea with quick chrome tests in `before_script` is better than removing `travis_retry`. I'll try that.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[.travis.yml, line 16 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-KaM0y0LI_uf9Vu8dzBc:-KaXWxauHFiKUCHxe8EL:b-42r2qa) ([raw file](https://github.com/vaadin/vaadin-element-skeleton/blob/850fb5f21c1c483b3cb564ddd9951319ff98c47c/.travis.yml#L16)):*
<details><summary><i>Previously, manolo (Manuel Carrasco Moñino) wrote…</i></summary><blockquote>

Oh, I thought travis stops when there is a non zero exit code.
But we still have options, like run fast tests in the `before_script` phase ?
</blockquote></details>

And this sounds like a good idea to me! I'll try your suggestion.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/12#-:-KaXXA28ttitS_EHBj7K:bn36lo1)*
